cms/wp/uploads/pull.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Wordpress Uploads Pull (rsync)
+ *
+ * @file
+ * @package wp-deployer
+ */
+
+namespace Deployer;
+
+task(
+	'wp:uploads:pull',
+	function() {
+		$upload_dir = 'web/app/uploads';
+		writeln( '<comment>> Receive remote uploads ... </comment>' );
```

Comment:
Is that extra `>` deliberate?

```suggestion
		writeln( '<comment> Receive remote uploads ... </comment>' );
```

(It might be in a few other tasks too)

Reply:
This is intentional, however was from a different author (previous wordpress deployer template v6). It's a stylistic choice of output by that author and not a syntax necessity.  I thought it added a nice little touch but am not apposed to removing it.

Reply:
I'm okay with it since it was intentional :)